### DISCUSSION TITLE
Http timeouts after setup

### DISCUSSION THREAD

**Discussion by jackyho-pb** [getredash/redash#7092] [Q&A]
I am trying to setup a redash instance using one of the pre-baked amis in this page (https://redash.io/help/open-source/setup/).  I chose t2.medium and also get assigned a public ip with security groups setup to whitelist http, https and ssh.  When I ssh into the EC2, I can see redash workers running; however, I always get a timeout if I try to http to the address.  May I know how to further debug this issue?

**Comment by justinclift** [comment]
Ahhhh.  Those AMIs are really old now.  We should update things to use newer ones at some point, though that's unlikely in the short term.

You'll probably be better off nuking that and running the official Redash setup.sh script here, probably on a Debian 12 VM: https://github.com/getredash/setup/

That'll need the same http/https/ssh security group access setup for it.
